study/hello-caqti-blocking/lib/repo/exec.ml

```diff
@@ -16,10 +16,32 @@ module Q = struct
     Caqti_type.(t2 int int ->! int)
     "SELECT ? + ?"
   [@@ocamlformat "disable"]
+
+  let mul =
+    Caqti_type.(t2 int int ->! int)
+    "SELECT ? * ?"
+  [@@ocamlformat "disable"]
 end
 
 (* Now we use our query definitions *)
 
-let add (module Conn : Caqti_blocking.CONNECTION) a b =
+(** Caqti uses "first-class modules" to manage database connections.
+  * Think of it like a simple value which you pass around and can't do anything with, until you "unpack"
+  * it, for Caqti's internal usage.
+  *
+  * In this example, we consume the [conn] first-class module by unpacking it into a normal module [Conn].
```

Comment:
"consume" may wrongly suggest that the operation cannot be repeated. What the unpacking does is just to make the interface available for use, with no dynamic effects.